command/operator_api_test.go

```diff
@@ -220,3 +223,74 @@ func TestOperatorAPICommand_ContentLength(t *testing.T) {
 		t.Fatalf("timed out waiting for request")
 	}
 }
+
+func makeSocketListener(t *testing.T) (net.Listener, string) {
+	td := os.TempDir() // testing.TempDir() on macOS makes paths that are too long
+	sPath := path.Join(td, t.Name()+".sock")
+	os.Remove(sPath) // git rid of stale ones now.
+
+	t.Cleanup(func() { os.Remove(sPath) })
```

Comment:
Just because the test runner won't clean this up and it might be polite to tidy up:

```suggestion
	t.Cleanup(func() { os.RemoveAll(td) })
```

Reply:
`os.TempDir()` gives the path to the operating system's default temp dir, which is why the test only deletes the socket file it makes.

Reply:
Oh, good catch. That'd have been bad :grinning: